### ISSUE TITLE
[Bug]: Moonlight can not display virtual display

### ISSUE THREAD

**Issue by SydneyWafflez** [VirtualDisplay/Virtual-Display-Driver#235]
### Which OS?

Win11 Pro

### Which release?

24H2

### Describe the bug

I am using Sunshine v0.23.1 on a Windows 11 PC (Running ReviOS), streaming to Moonlight on a Nvidia Shield. I verified the output name via the cmd tool included in sunshine. When I select "Desktop" in Moonlight, it tries to connect, sits on a black screen for 4-5 seconds, then shows a CONNECTION TERMINATED: No video received from host error.
![image](https://github.com/user-attachments/assets/bb14eed8-df6c-4f2d-8cb6-f042c7da0646)
![image](https://github.com/user-attachments/assets/33f60672-bf2b-4233-9d75-9704bad1c209)

### Steps to reproduce

1) Set up 1 virtual display
2) Use dxgi tool to get display name
3) Set display name in Sunshine and restart
4) Go to device on Moonlight, and select Desktop
5) Error

### Expected behavior

Successful display of the 4th monitor

### Contact Details

Email: sydney@morrison.gg

**Comment by Cactism** [comment]
Can confirm on my setup as well; I migrated from [24.10.27](https://github.com/VirtualDisplay/Virtual-Display-Driver/releases/tag/24.10.27) which was working perfectly before moving to the [latest stable](https://github.com/VirtualDisplay/Virtual-Display-Driver/releases/tag/24.12.24) release

**Comment by bud3699** [comment]
If i build a custom one, do you guy mind testing it out? I have a feeling what the issue is but not certain

**Comment by Cactism** [comment]
@bud3699 Yes, I'm willing to try it out!

**Comment by SydneyWafflez** [comment]
> If i build a custom one, do you guy mind testing it out? I have a feeling what the issue is but not certain

Yep, would be willing!

**Comment by mleone12** [comment]
@bud3699 I am having the same issue, when I use the older driver it works fine I can close my laptop screen and connect with moonlight to sunshine on my laptop using the virtual display. Could I get a copy of the custom one as well? The new driver with the companion is awesome! Great work!

**Comment by YinHeng89** [comment]
Yes, I also encountered this problem with the new version of mttvdd,If you turn off the real monitor and switch to the virtual monitor while streaming, it will not load correctly.

**Comment by Anankke** [comment]
I am having the same issue here...

**Comment by skyerus** [comment]
Also having this issue

**Comment by fuomag9** [comment]
Same issue, this only happens if I update windows! If I restore to an older snapshot pre-update it will work fine again.
